txstratum/protocol.py

```diff
@@ -234,6 +249,17 @@ def start_periodic_tasks(self) -> None:
             )
             asyncio.ensure_future(self.messages_timeout_task.start())
 
+        if self.rtt_estimator_task is None:
+            # Start rtt estimator periodic task.
+            self.rtt_estimator_task = Periodic(
+                self.rtt_estimator, self.RTT_ESTIMATOR_INTERVAL
+            )
+            asyncio.ensure_future(self.rtt_estimator_task.start())
+
+    async def rtt_estimator(self) -> None:
+        """Period task to request the miner version to calculate the rtt."""
+        self.send_and_track_request("client.get_version", None)
```

Comment:
Should we check whether the last request was fulfilled? I guess we shouldn't have two requests waiting for reply.

Reply:
The interval of this RTT estimator is only 5 seconds, while the timeout of the message sent to the miners is 30 seconds.

This opens a room for this possibility to happen.

Reply:
Done https://github.com/HathorNetwork/tx-mining-service/pull/114/commits/413e9e2f80260e45b8f23f3b876d6f975599ce69

Reply:
Great!